test/features/pagecontent/save_api.js

```diff
@@ -251,10 +272,28 @@ describe('page save api', function() {
         });
     });
 
+    it('should forbid external usage', function() {
```

Comment:
I don't see how this test should pass. So, the idea is that RESTBase should allow page saves for internal clients and fail for others. But ... in test mode, there are no external clients, only a local one. Hence, if this test is passing, something's wrong in the code, as `req.socket.remoteAddress` in this case must be equal to `127.0.0.1`